app/views/spree/checkout/payment/_gateway.html.erb

```diff
@@ -1,35 +1,43 @@
-<% param_prefix = "payment_source[#{payment_method.id}]" %>
+<div class="gateway-method">
```

Comment:
Again, I think we should find a better name for this class.

Reply:
This is a legacy file that will probably get removed soon, I'd leave this for another PR.